README.md

```diff
@@ -285,271 +287,209 @@ puts sd.count_entity_total()
 }
 ```
 
-### `count_entity(json_data, test = False)`
+### `count_entity(json_data)`
 Count the number of entities attending the given query. This method corresponds to a [POST request at /query/count/entity](http://panel.slicingdice.com/docs/#api-details-api-endpoints-post-query-count-entity).
 
 #### Request example
 
 ```ruby
 from pyslicer import SlicingDice
-sd = SlicingDice('MASTER_OR_READ_API_KEY')
+client = SlicingDice.new(master_key: "API_KEY", uses_test_endpoint: false)
 query = {
-    'users-from-ny-or-ca': [
+    'corolla-or-fit': [
         {
-            'state': {
-                'equals': 'NY'
+            'car-model': {
+                'equals': 'toyota corolla'
```

Comment:
You should be consistent, in some place you are using `=>` to on pair `value:key`. in another `:`. Choose one, and apply in all `Request examples`